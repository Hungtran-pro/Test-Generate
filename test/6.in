33
93
82
61
61
101
85
68
97
80
62
87
88
81
60
60
70
86
86
111
65
105
61
60
72
101
65
107
101
78
100
77
78
81
