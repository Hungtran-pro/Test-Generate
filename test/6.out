482361088
639909956
312450027
312450027
351513474
100543565
599807473
198734243
991943513
217689433
241802082
142382092
459405516
266177107
266177107
816064847
980141720
980141720
466486836
562992784
676689719
312450027
266177107
170325148
351513474
562992784
472606716
351513474
116509860
742820472
216191719
116509860
459405516
